#!/home/davidyam/anaconda3/envs/tele_bot/bin/python3.6
import sys
from lib2to3.main import main

sys.exit(main("lib2to3.fixes"))
